libs/execution_context/ebpf_maps.h

```diff
@@ -148,26 +148,26 @@ extern "C"
      * program returned holds a reference that the caller is responsible for
      * releasing.
      *
-     * @param[in] map Map to search.
+     * @param[in,out] map Map to search.
```

Comment:
If it's in/out then maybe
```suggestion
     * @param[in,out] map Map to search and update metadata in.
```